catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -52,6 +55,27 @@
     Processor systemSubjectBinder = new SystemSubjectBinder();
 
     /**
+     * Policy for polling the 'content' CamelComponent. Specifies the number of times any CDM will
+     * poll and the number of seconds any CDM will wait between consecutive polls.
+     */
+    RetryPolicy getContentComponentRetryPolicy() {
```

Comment:
Instead of using a factory method, why not inject the `RetryPolicy` in the constructor or a setter? Would be a little easier to deal with in the tests.

And we shouldn't use `CDM` anywhere. It's not a well-known acronym.